src/icons/fileIcons.ts

```diff
@@ -1626,8 +1626,11 @@ export const fileIcons: FileIcons = {
         'gdns',
         'gdnlib',
         'gdshader',
+        'gdshaderinc',
         'gdextension',
+        'import',
```

Comment:
Please remove the "import" file extension. You might imagine that there are other developers using this extension as well who have projects with files called "myfile.import" but they are not using Godot. We should avoid using such common file extensions or file names which could be used in other contexts as well. Then the icon might be confusing.

Reply:
I actually went back-and-forth on adding this, but ultimately did because the only references to that extension I could find were all related to Godot. That said, yeah it is an oddly common name for what is otherwise a consistently unique naming scheme. An Icon Pack for a single name would also be excessive, so this might be local-override territory